### PR TITLE
Support dynamically set log level for kube-proxy

### PR DESCRIPTION
**Pull request by borgerli**
For now, the kube-apiserver, kubelet and kube-scheduler components all support dynamically setting log level.

This function is really helpful and convenient for troubleshooting with production clusters, especially for cases that problems disappear if we change --v parameter and then restart the component.

This PR is for kube-proxy.

### CODE REVIEW

File: pkg/proxy/healthcheck/proxier_health.go [hunk 1]
```diff
@@ -92,6 +94,7 @@ func (hs *proxierHealthServer) QueuedUpdate() {
 func (hs *proxierHealthServer) Run() error {
 	serveMux := http.NewServeMux()
 	serveMux.Handle("/healthz", healthzHandler{hs: hs})
+	serveMux.Handle("/debug/flags/v", routes.StringFlagPutHandler(logs.GlogSetter))
```
Comment (aojea) [inline pkg/proxy/healthcheck/proxier_health.go:97]: kube-proxy has a metrics and a healthcheck server

https://github.com/kubernetes/kubernetes/blob/661eae7c7c8226a3f7bed3a24acf1c22861477b7/cmd/kube-proxy/app/server.go#L666-L670

IMHO the health server doesn't seems the right place , should we open an issue first to discuss this?

Comment (borgerli) [reply pkg/proxy/healthcheck/proxier_health.go:97]: @aojea Thanks for review.  

The set-log-level handler needs to be installed to metrics server and controlled by 'profiling' flag. 

I just found an issue related to this flag: #98302. I'll close this PR for now, and request another PR when issue #98302 got fixed.

**Pull request closed without merging**